src/masternode/masternode-payments.cpp

```diff
@@ -278,7 +275,7 @@ std::map<int, std::string> GetRequiredPaymentsStrings(int nStartHeight, int nEnd
         auto projection = deterministicMNManager->GetListAtChainTip().GetProjectedMNPayees(nEndHeight - nChainTipHeight);
         for (size_t i = 0; i < projection.size(); i++) {
             auto payee = projection[i];
-            int h = nChainTipHeight + 1 + i;
+            uint h = nChainTipHeight + 1 + i;
```

Comment:
? this should be reverted

Reply:
It was partially intended. Because i is size_t so h should be unsigned, however using uint is undefined behavior it seems, so changed to size_t. I guess I could use auto too